src/components/Prompt/Prompt.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+@keyframes Prompt_open {
+    0% {
+        opacity: 0;
+        transform: translateY(100%);
+    }
+    100% {
+        opacity: 1;
+        transform: translateY(0%);
+    }
+}
+
+@keyframes Prompt_close {
+    0% {
+        opacity: 1;
+        transform: translateY(0%);
+    }
+    100% {
+        opacity: 0;
+        transform: translateY(100%);
+    }
+}
+
+.Prompt {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    overflow: hidden; // prevent scrollbar
+    position: fixed;
+    bottom: 0;
+
+    &:not(.Prompt_mounted) {
+        display: none;
+    }
+
+    &__content {
+        $duration: $animationDuration * 2;
+        @extend %shadow;
+        display: flex;
+        flex-flow: row wrap;
+        gap: $indentXS;
+        align-items: center;
+        margin: $indentSM;
+        padding: $indentXS $indentS;
+        border-radius: $borderRadius;
+        background-color: var(--yc-color-base-float);
+        font-size: 1.25em;
+
+        .Prompt_open > & {
+            opacity: 0;
+            transform: translateY(100%);
+            animation: Prompt_open $duration forwards;
+        }
+
+        .Prompt_close > & {
+            opacity: 1;
+            transform: translateY(0%);
+            animation: Prompt_close $duration forwards;
+        }
+    }
+}
+
+.Prompt_close {
+    // let other content receive Pointer events
```

Comment:
I do not think this comment is should be there. We actually leave comment in a code only if there is a controversial solution or a temporary crutch.

Here is absolutely clear what `pointer-events: none;` serves for